packages/core/src/defer/interfaces.ts

```diff
@@ -7,7 +7,19 @@
  */
 
 import type {Provider} from '../di/interface/provider';
+import {LContainer} from '../render3/interfaces/container';
```

Comment:
nit: this should help to resolve the remaining circular dep:
```suggestion
import type {LContainer} from '../render3/interfaces/container';
```